src/deepsparse/engine.py

```diff
@@ -144,15 +156,21 @@ def __init__(
         batch_size: int,
         num_cores: int,
         num_sockets: int = None,
+        scheduler: str = None,
```

Comment:
Could we make scheduler an enum so users have to pick one of those values? We can set the enum names to the values needed by the backend and have the call into deepsparse_engine pass the enum.value

Reply:
We have an enum already inside the engine that this immediately translates to. 
It looks like we could wrap it with pybind and expose it, I'll give it a try.
https://stackoverflow.com/questions/47893832/pybind11-global-level-enum